src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_chatbot.py

```diff
@@ -8,8 +8,9 @@
 from aws_cdk import (
     aws_codestarnotifications as cp_notifications,
     aws_codepipeline as codepipeline,
-    core,
+    Stack
```

Comment:
Can you add a comma at the end, so in the future if we add more requirements we don't need to modify this line too?

```suggestion
    Stack,
```